content/php/concepts/date-functions/terms/date-diff/date-diff.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'date_diff()'
+Description: 'Takes two DateTime objects, calculates the difference, and returns a DateInterval object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+  - 'Objects'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `date_diff()` function takes two `DateTime` objects as parameters and determines the difference in time between them. An optional boolean parameter can be set to `true` to return the absolute value, ensuring that the result is positive. The function returns a `DateInterval` object that can be formatted as needed.
+
+## Syntax
+
+```pseudo
+date_diff(
+    DateTimeInterface $dateOne,
+    DateTimeInterface $dateTwo
+    bool $absolute=false
+): DateInterval
+```
+
+The `$dateOne` and `$dateTwo` parameters are `DateTime` objects as returned by [date_create()](https://www.codecademy.com/resources/docs/php/date-functions/date-create). Because the returned value is a `DateInterval` object, [date_format()](https://www.codecademy.com/resources/docs/php/date-functions/date-format) cannot be used to format the result. The `DateInterval::format()` function can be used for formatting using the characters below. The formatting characters must be prefixed with a percent sign (`%`).
```

Comment:
```suggestion
The `$dateOne` and `$dateTwo` parameters are `DateTime` objects as returned by [`date_create()`](https://www.codecademy.com/resources/docs/php/date-functions/date-create). Because the returned value is a `DateInterval` object, [`date_format()`](https://www.codecademy.com/resources/docs/php/date-functions/date-format) cannot be used to format the result. The `DateInterval::format()` function can be used for formatting using the characters below. The formatting characters must be prefixed with a percent sign (`%`).
```

Reply:
Added backticks to linked functions.